be/src/common/config.h

```diff
@@ -414,8 +414,14 @@ CONF_mInt32(streaming_load_rpc_max_alive_time_sec, "1200");
 // the timeout of a rpc to open the tablet writer in remote BE.
 // short operation time, can set a short timeout
 CONF_Int32(tablet_writer_open_rpc_timeout_sec, "60");
+// the timeout of a rpc to open the partition in remote BE.
```

Comment:
please add description for these flags in `docs/zh-CN/docs/admin-manual/config/be-config.md` and `docs/en/docs/admin-manual/config/be-config.md`